compiler/plugins/target/AMD-AIE/iree-amd-aie/IR/AMDAIEOps.td

```diff
@@ -1053,7 +1053,8 @@ def AMDAIE_NpuWriteBdOp: AMDAIE_Op<"npu.write_bd"> {
     ```
   }];
   let arguments = (
-    ins UI32Attr:$col,
+    ins Optional<Index>:$connection,
```

Comment:
Maybe add some documentation to the op description about the need for the connection op.